Hello Beloved in Christ.

```diff
@@ -0,0 +1,32 @@
+
+Hello Beloved in Christ.
+
+I apologize for this intrusion , my name is Metthey Françoise
+originally from France . I can contact you in this way because I want
+to do something very important . This will seem a little suspect true
+that you do not know me and I also do not know you . In fact, i am
+suffering from cancer of leukemia which the doctors has confirmed to
+me that i will not last long due to the last diagnoses result which i
+received right here at the Hospital in  Ivory Coast where i am been
+treated right now., I have this disease for more than 4 years. I am a
```

Comment:
This is poor grammar, please fix. I don't think we can accept this in its current form.